web/docs/migrate-from-0-11-to-0-12.md

```diff
@@ -367,7 +367,7 @@ module.exports = {
 </TabItem>
 </Tabs>
 
-### Default Server Dockerfile Changed
+### Migrating a custom Dockerfile 
```

Comment:
@sodic there was a reason why it was named that way -> because you might not need to migrate a custom Dockerfile, but update your deployment mechanism.
What really changed is not default dockerfile, but how server is being built / prepared for deployment. We had some people asking how to get it working on bare metal, and what they did was copy the instructions from the Dockerfile. So for them, they don't need to migate a custom dockerfile, they need toupdate their command on their server or wherever.

We went with "Default Server Dockerfile Changed" as a compromise between correctnes and recognizability. If you based your custom instructions based on the default dockerfile, which is really the only source of truth on how server should be deployed, then this should pick your interest. Migrating a custom Dockerfile on the other hand sounds like I don't have to take a look at it if I don't have a custom dockerfile, which is not correct.

I don't mind "sneaking" in changes like this in the PR that has nothing to do with it, although in theory they should be a separate PR -> it can be too much bearocracy, I get that. But if doing so, it could be useful to mark it with a comment for reviewers, so we don't miss it, because something like this is super easy to miss.

Also, Chesterton's fence would be a good rule that could have helped here.